img/b/brilliant.org.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100"><path id="a" fill="#535353" d="M50 0A50 50 0 0 0 0 50a50 50 0 0 0 50 50V0zm-1.3 2.6v1a84.4 84.4 0 0 0-22.6 14.2A119 119 0 0 1 32 6a47.2 47.2 0 0 1 16.6-3.5zm0 3.7v7c-1.6.1-10.2 1-21 6.4a82.3 82.3 0 0 1 21-13.4zM28 8c-2.2 4.1-4.1 8.2-5.8 12.3-1-2.3-1.8-4.6-2.5-6.7A47.7 47.7 0 0 1 28 8zm-10.4 7.4a102 102 0 0 0 3 7.4c-4-.5-7-.6-9-.7a47 47 0 0 1 6-6.7zm31.1.4v13.8c-8-3.2-16-5-22.8-6.1a63.3 63.3 0 0 1 22.8-7.7zM10 24.6c1 0 4.2 0 8.8.5A59.2 59.2 0 0 0 7 46.4a15.5 15.5 0 0 1-3.7-5A47.1 47.1 0 0 1 10 24.7zM25 26a105.2 105.2 0 0 1 21.6 5.6A190 190 0 0 0 33.1 41c-2.9-4.9-5.8-10-8.3-15.2zm-2.6 1C25 32.4 28 37.7 31 42.7c-3.8 3.1-7.9 6.7-11.8 10.6A66 66 0 0 1 22.4 27zm-3 1.3a67.2 67.2 0 0 0-2.8 25.2A59.4 59.4 0 0 1 9 48.3a56 56 0 0 1 10.4-20zm29.2 4.8v31.2a312.6 312.6 0 0 1-14.2-21C40.6 38.5 46 35 48.7 33.1zM32.5 45a314.8 314.8 0 0 0 14.4 21.3 159.8 159.8 0 0 1-26.3-10.6c4-4 8-7.5 11.9-10.7zm-29.7.6a25.2 25.2 0 0 0 3.4 3.7c-1 4.4-1.7 9.2-1.7 14.3a47.3 47.3 0 0 1-1.7-18zM8.4 51a68.8 68.8 0 0 0 8 5.2 88.5 88.5 0 0 0-9.2 11.5A56 56 0 0 1 8.4 51zM20 58a156 156 0 0 0 25.3 10.3c-3.8 1-10.5 2-21.4 2.3l-.5-1.1A53.8 53.8 0 0 1 20 58zm-2.5.8A56.4 56.4 0 0 0 21 70.6v.1c-3.7 0-8 0-12.6-.3a81.2 81.2 0 0 1 9-11.5zm29 11.8c-2.4 3.1-6.9 8.2-13.4 14a56.2 56.2 0 0 1-8-11.5 94.9 94.9 0 0 0 21.4-2.5zm2.2 1.4v14.4a77 77 0 0 1-12.7-1A105.8 105.8 0 0 0 48.7 72zM9.1 73l11.3.3h2A58.5 58.5 0 0 0 29 83.8c-7-1.9-14.4-5.2-20-11zm3.7 6.4A51.9 51.9 0 0 0 30.3 87a129.5 129.5 0 0 1-4.7 3.8 48 48 0 0 1-12.8-11.3zm20 8.8A57.2 57.2 0 0 0 43.6 97 47 47 0 0 1 28 92a130.6 130.6 0 0 0 4.8-3.8zm3.6 0c5.1.7 9.4.9 11.7.9h.6v7.8a47 47 0 0 1-12.3-8.7z"/><use xlink:href="#a"/><use xlink:href="#a" transform="matrix(-1 0 0 1 100 0)"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100"><path id="a" fill="#1f1f1f" d="M50 0A50 50 0 0 0 0 50a50 50 0 0 0 50 50V0zm-1.3 2.6v1a84.4 84.4 0 0 0-22.6 14.2A119 119 0 0 1 32 6a47.2 47.2 0 0 1 16.6-3.5zm0 3.7v7c-1.6.1-10.2 1-21 6.4a82.3 82.3 0 0 1 21-13.4zM28 8c-2.2 4.1-4.1 8.2-5.8 12.3-1-2.3-1.8-4.6-2.5-6.7A47.7 47.7 0 0 1 28 8zm-10.4 7.4a102 102 0 0 0 3 7.4c-4-.5-7-.6-9-.7a47 47 0 0 1 6-6.7zm31.1.4v13.8c-8-3.2-16-5-22.8-6.1a63.3 63.3 0 0 1 22.8-7.7zM10 24.6c1 0 4.2 0 8.8.5A59.2 59.2 0 0 0 7 46.4a15.5 15.5 0 0 1-3.7-5A47.1 47.1 0 0 1 10 24.7zM25 26a105.2 105.2 0 0 1 21.6 5.6A190 190 0 0 0 33.1 41c-2.9-4.9-5.8-10-8.3-15.2zm-2.6 1C25 32.4 28 37.7 31 42.7c-3.8 3.1-7.9 6.7-11.8 10.6A66 66 0 0 1 22.4 27zm-3 1.3a67.2 67.2 0 0 0-2.8 25.2A59.4 59.4 0 0 1 9 48.3a56 56 0 0 1 10.4-20zm29.2 4.8v31.2a312.6 312.6 0 0 1-14.2-21C40.6 38.5 46 35 48.7 33.1zM32.5 45a314.8 314.8 0 0 0 14.4 21.3 159.8 159.8 0 0 1-26.3-10.6c4-4 8-7.5 11.9-10.7zm-29.7.6a25.2 25.2 0 0 0 3.4 3.7c-1 4.4-1.7 9.2-1.7 14.3a47.3 47.3 0 0 1-1.7-18zM8.4 51a68.8 68.8 0 0 0 8 5.2 88.5 88.5 0 0 0-9.2 11.5A56 56 0 0 1 8.4 51zM20 58a156 156 0 0 0 25.3 10.3c-3.8 1-10.5 2-21.4 2.3l-.5-1.1A53.8 53.8 0 0 1 20 58zm-2.5.8A56.4 56.4 0 0 0 21 70.6v.1c-3.7 0-8 0-12.6-.3a81.2 81.2 0 0 1 9-11.5zm29 11.8c-2.4 3.1-6.9 8.2-13.4 14a56.2 56.2 0 0 1-8-11.5 94.9 94.9 0 0 0 21.4-2.5zm2.2 1.4v14.4a77 77 0 0 1-12.7-1A105.8 105.8 0 0 0 48.7 72zM9.1 73l11.3.3h2A58.5 58.5 0 0 0 29 83.8c-7-1.9-14.4-5.2-20-11zm3.7 6.4A51.9 51.9 0 0 0 30.3 87a129.5 129.5 0 0 1-4.7 3.8 48 48 0 0 1-12.8-11.3zm20 8.8A57.2 57.2 0 0 0 43.6 97 47 47 0 0 1 28 92a130.6 130.6 0 0 0 4.8-3.8zm3.6 0c5.1.7 9.4.9 11.7.9h.6v7.8a47 47 0 0 1-12.3-8.7z"/><use xlink:href="#a"/><use xlink:href="#a" transform="matrix(-1 0 0 1 100 0)"/></svg>
```
The colour looks slightly off to me

Reply:
FWIW there is an official SVG from brilliant.org:
https://upload.wikimedia.org/wikipedia/en/8/81/Brilliant_Logo.svg

The fill is #333333

Reply:
I used the logo from https://brilliant.org/site_media/version-0/images/about/brilliant-logo.svg (how's that for an official one :smile:) which has `#535353` as the stroke color. So there seem to be several versions.

Reply:
OK, probably fine then.

Reply:
Hah! yeah I found it as well probably worth updating the wikipedia one to use the official logo